hack/assets/assets.go

```diff
@@ -101,9 +100,12 @@ func GetSupportedDistros() []string {
 
 // GetImages returns a list of images based on the given parameters
 func GetImages(cleanTag string, latest bool, optional bool, kubernetesVersion string, kubernetesDistro string) []string {
-	var images []string
+	images := []string{"ghcr.io/loft-sh/vcluster-oss:" + cleanTag}
 	if !optional {
-		images = GetVclusterImages(cleanTag)
+		if latest {
```

Comment:
I would move this logic to the `GetVclusterImages()` func to make the `GetImages` func cleaner and and easier to read

Reply:
Yep! Sure!